conda-forge.yml

```
@@ -1 +1,9 @@
+build_platform:
+  osx_arm64: osx_64
```

If cross-compiling is already working, try to use it for all arches instead of full emulation.
```suggestion
  osx_arm64: osx_64
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```